uint16 prod_id
float32 yaw
float32 pitch
float32 roll
float32 TEMP_OUT
float32 X_GYRO_OUT
float32 Y_GYRO_OUT
float32 Z_GYRO_OUT
float32 X_ACCL_OUT
float32 Y_ACCL_OUT
float32 Z_ACCL_OUT
float32 X_MAGN_OUT
float32 Y_MAGN_OUT
float32 Z_MAGN_OUT
float32 BAROM_OUT


